### PR TITLE
Some Options matter now

### PR DESCRIPTION
**Pull request by Couls**
Capitilized name and made some answers matter

### CODE REVIEW

**Review by Sir-Weebz-A-Lot — APPROVED** [review]

**Comment by Couls** [comment]
waiting for osmos review before merging

**Review by osmos1 — COMMENTED** [review]
dass the old code i already fixed this

**Pull request merged**
